src/main/java/seedu/address/model/activity/Activity.java

```diff
@@ -75,6 +91,7 @@ public void invite(Person ... people) {
      */
     public void disinvite(Person ... people) {
         // haven't implemented what if list does not contain that specific person
+        // TODO: also care about transfermatrix
```

Comment:
Ideally this should be done in disinvite command. Validations should be done in command class.